test/identifier/multiple-statement.spec.ts

```diff
@@ -170,5 +126,113 @@ describe('identifier', () => {
 
       expect(actual).to.eql(expected);
     });
+
+    describe('identifying multiple statements with CTEs', () => {
+      it('should able to detect queries with a CTE in middle query', () => {
```

Comment:
This was a pre-existing test, just grouped it with other CTE tests in describe block. No change to the actual test itself.